src/soep_preparation/utilities/general.py

```diff
@@ -51,6 +53,31 @@ def get_data_file_names(
     return script_names
 
 
+def get_relevant_column_names(script_path: Path) -> list[str]:
+    """Get relevant column names from the cleaning script.
+
+    Args:
+        script_path: The path to the cleaning script.
+
+    Returns:
+        A list of relevant column names.
+    """
+    module = load_module(script_path)
+    function_with_docstring = inspect.getsource(module.clean)
+    # Remove the docstring, if existent.
+    function_content = re.sub(
+        r'""".*?"""|\'\'\'.*?\'\'\'',
+        "",
+        function_with_docstring,
+        flags=re.DOTALL,
```

Comment:
```suggestion
    # Remove the docstring, if existent.
    function_source = re.sub(
        r'""".*?"""|\'\'\'.*?\'\'\'',
        "",
        inspect.getsource(module.clean),
        flags=re.DOTALL,
```

No need to get distracted by having two versions around.